analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -129,7 +129,7 @@ def define_trijet_mass(df: ROOT.RDataFrame) -> ROOT.RDataFrame:
     """Add the trijet_mass observable to the dataframe after applying the appropriate selections."""
 
     # First, select events with at least 2 b-tagged jets
-    df = df.Filter("Sum(Jet_btagCSVV2[Jet_pt_mask]>0.5)>1")
+    df = df.Filter("Sum(Jet_btagCSVV2_masked>0.5)>1")
```

Comment:
sorry, but since we are touching this line let's make it more readable:
```suggestion
    df = df.Filter("Sum(Jet_btagCSVV2_masked > 0.5) > 1")
```